js/event.js

```diff
@@ -197,6 +197,8 @@
                     return evt;
                 }
             }
+			evt.x = evt.pageX;
+			evt.y = evt.pageY;
```

Comment:
We don't need to do this, and it didn't seem to work. We'll just use evt.pageX/evt.pageY